cmd/admin-obd.go

```diff
@@ -0,0 +1,399 @@
+/*
+ * MinIO Client (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/klauspost/compress/gzip"
+	"github.com/minio/cli"
+	cjson "github.com/minio/mc/pkg/colorjson"
+	"github.com/minio/mc/pkg/probe"
+	"github.com/minio/minio/pkg/console"
+	"github.com/minio/minio/pkg/madmin"
+)
+
+var adminOBDFlags = []cli.Flag{
+	OBDDataTypeFlag{
+		Name:   "tests",
+		Usage:  "choose OBD tests to run [" + options.String() + "]",
+		Value:  nil,
+		EnvVar: "MINIO_OBD_DATA",
```

Comment:
For `mc` we need to prefix MC_ not MINIO_ @wlan0 - is this confirmed with @abperiasamy ?

Reply:
No, it is not confirmed. I think it should be MC